pkgs/applications/kde/akonadi/default.nix

```diff
@@ -20,10 +23,10 @@ mkDerivation {
   propagatedBuildInputs = [ boost kitemmodels ];
   outputs = [ "out" "dev" ];
   NIX_CFLAGS_COMPILE = [
-    ''-DNIXPKGS_MYSQL_MYSQLD="${lib.getBin mysql}/bin/mysqld"''
-    ''-DNIXPKGS_MYSQL_MYSQLADMIN="${lib.getBin mysql}/bin/mysqladmin"''
-    ''-DNIXPKGS_MYSQL_MYSQL_INSTALL_DB="${lib.getBin mysql}/bin/mysql_install_db"''
-    ''-DNIXPKGS_MYSQL_MYSQLCHECK="${lib.getBin mysql}/bin/mysqlcheck"''
+    ''-DNIXPKGS_MYSQL_MYSQLD="${mysql}/bin/mysqld"''
+    ''-DNIXPKGS_MYSQL_MYSQLADMIN="${mysql}/bin/mysqladmin"''
+    ''-DNIXPKGS_MYSQL_MYSQL_INSTALL_DB="${mysql}/bin/mysql_install_db"''
+    ''-DNIXPKGS_MYSQL_MYSQLCHECK="${mysql}/bin/mysqlcheck"''
```

Comment:
I won't include this one as it might very well be that we add a bin output to mysql in the future.

Reply:
ok, rebased